README.md

```diff
@@ -47,6 +47,17 @@ second, it should require about 290 million years.
 See src/stockpile.clj and test/stockpile_test.clj for the API
 documentation and sample usage.
 
+## Testing
+
+As expected, "lein test" will run the test suite, but there are some
+additional tests can only be run if STOCKPILE\_TINY\_TEST\_FS is set
```

Comment:
Wyatt has pointed out that \_ doesn't render quite right on our website (I know this isn't on the website but it might be nice to stick to the convention of `\`STOCKPILE_TINY_TEST_FS\``